lib/location/cloud_service/cloud_service_here_rest.c

```diff
@@ -89,11 +91,6 @@ BUILD_ASSERT(sizeof(CONFIG_LOCATION_SERVICE_HERE_API_KEY) > 1,
 		"\"rsrq\":%.1f"						\
 	"}"
 
-BUILD_ASSERT(sizeof(HOSTNAME) > 1, "Hostname must be configured");
-
-static char body[HTTP_BODY_SIZE];
-static char neighbors[NEIGHBOR_BUFFER_SIZE];
```

Comment:
There are quite a lot of changes in HERE cloud service implementation as Wi-Fi and cellular had been made in completely different way.
Cellular used static buffers and sprintf to insert missing dynamic information into prefefined json templates.
At the same time, WiFi has been made using cJSON library and that was used to get the HTTP request body.
I haven't changed cellular to use cJSON but I've removed majority of the static buffers to free SRAM.

Reply:
Changing of cellular Here to be using cjson lib was supposed to be a task of Jira NCSDK-13916. But anyways, that is really needed when GCI support is included for that.

Reply:
Good, I'm more than happy to leave cJSON conversion for later...